### PR TITLE
Added dlog, gogga, libpb, jstruct

### PR DESCRIPTION
**Pull request by deavmi**
Added a few libraries.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -329,6 +329,7 @@
 * [painlessjson](https://github.com/BlackEdder/painlessjson) - Convert between D types and std.json.
 * [std.data.json](https://github.com/dlang-community/std_data_json) - Phobos candidate for JSON serialization (based on Vibed)
 * [asdf](https://github.com/libmir/asdf) - Cache oriented string based JSON representation for fast read & writes and serialisation.
+* [jstruct](https://github.com/Hax-io/jstruct) - JSONify a struct and vice-versa
```
Comment (WebFreak001) [inline README.md:332]: I'm up for adding new projects, but this project has 0 stars on github and 16 downloads total on DUB. The functionality is also already provided by more popular packages such as painlessjson or mir-ion (which is also active in development and provides a standard (de)serialization framework that also works with a bunch of safety attributes and DIPs)

I don't think we should add this to the list here (it also only has 10 commits and only a few days of activity 2 months ago)

**Review by WebFreak001 — CHANGES_REQUESTED** [review]
the other projects don't look massively used as well, but we can include them as they don't really overlap with other things we have in the list.

File: README.md (hunk 1 above)
Comment (deavmi) [reply README.md:332]: Will update tomorrow evening 🤙👍

**Comment by deavmi** [comment]
Thanks!

**Pull request merged**